site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+
+* Record the breach in metrics 
+* Send the user an email
+* Require the user to change their password
+
+With any of these options, you can subscribe to a webhook notifying you of the password breach event.
+
+==== Recording the Breach 
+
+Recording the breach and optionally firing a webhook is a good fit for two scenarios. If you're exploring breached password detection and unsure how it will affect your end users, you can gauge how many of them have compromised passwords by using this setting. 
+
+Another reason to use this option is if you have link:../events-webhooks/[webhooks] set up. You may handle a breached password in custom code, rather than using any of the default FusionAuth options. Listening for the webhook lets you handle this event in any way you desire. You could:
+
+* Automatically run a security audit on systems to which the user with the compromised credentials has access
+* Lock the account and send an SMS notification
+* Flip a flag on their account to more closely monitor it for suspicious activity in the future
+
+All of these require custom code on your end, of course, but the webhook can trigger any custom action.
+
+==== Sending an Email
+
+Sending the user an email lets the user know that their account could be compromised, but doesn't force them to take any action.
+
+This email is an link:../email-templates/[email template] which can be customized and localized similar to any other template. You'll typically include a link to reset their password, but you may want to include other information as well. Below is the default HTML template.
+
+.HTML Breached Password Email Template
+[source,text]
+----
+include::docs/src/email/breached-password.html[]
+----
+
+Please see the link:/docs/v1/tech/email-templates/email-templates#breached-password[email template documentation] for more information, including available template variables.
+
+==== Requiring a Password Change
+
+The final option for handling breached passwords at user sign in is to require a password change. Once a password has been checked and found to be breached, the account is marked in the system as requiring a changed password. The user will be forced to change their password whenever they attempt to login. Be aware that changing the 'On login' setting to a different option will have no effect on accounts so marked.
+
+When this option is enabled, if you are using the FusionAuth templates, the page displayed to the user after they login with a compromised password is is generated by the "OAuth Change password form" template. As with any other template, you can customize it, including look and feel and any messages displayed. Please check out the link:../email-templates/[themes documentation] for more information.
```

Comment:
Suggestion: 

> ... if you are using a FusionAuth themed login, the page page ...

We have templates for Email and Themes, and perhaps in the future SMS, so being more specific than templates may be helpful.